qiskit_ibm_runtime/sampler.py

```diff
@@ -257,12 +259,33 @@ def _call(
         combined = self.options._merge_options(run_options)
         inputs.update(Options._get_program_inputs(combined))
 
-        return self._session.run(
+        raw_result = self._session.run(
             program_id=self._PROGRAM_ID,
             inputs=inputs,
             options=Options._get_runtime_options(combined),
             result_decoder=SamplerResultDecoder,
         ).result()
+        quasi_dists = []
+        for quasi, meta in zip(raw_result["quasi_dists"], raw_result["metadata"]):
+            shots = meta.get("shots", None)
+            overhead = meta.get("readout_mitigation_overhead", None)
+
+            if shots is None or shots == 0 or overhead is None:
+                stddev = None
+            else:
+                # M3 mitigation overhead is gamma^2
+                # https://github.com/Qiskit-Partners/mthree/blob/423d7e83a12491c59c9f58af46b75891bc622949/mthree/mitigation.py#L457
+                #
+                # QuasiDistribution stddev_upper_bound is gamma / sqrt(shots)
+                # https://github.com/Qiskit/qiskit-terra/blob/ff267b5de8b83aef86e2c9ac6c7f918f58500505/qiskit/result/mitigation/local_readout_mitigator.py#L288
+                stddev = sqrt(overhead / shots)
```

Comment:
```suggestion
            if not shots:
                shots = float('inf')
            if overhead is None:
                overhead = 1.0
            # M3 mitigation overhead is gamma^2
            # https://github.com/Qiskit-Partners/mthree/blob/423d7e83a12491c59c9f58af46b75891bc622949/mthree/mitigation.py#L457
            #
            # QuasiDistribution stddev_upper_bound is gamma / sqrt(shots)
            # https://github.com/Qiskit/qiskit-terra/blob/ff267b5de8b83aef86e2c9ac6c7f918f58500505/qiskit/result/mitigation/local_readout_mitigator.py#L288
            stddev = sqrt(overhead / shots)
```

Reply:
Thank you for the suggestion. I can even simplify this approach.

Reply:
See d52fa5a